### PR TITLE
Add activity script

### PR DESCRIPTION
**Pull request by klieret**
Recently (#671) we changed the way we add training schools to the list at https://hepsoftwarefoundation.org/Schools/events.html as discussed in #651 .

This adds a small python script with a command line interface to more conveniently add a new event to the yml database and checks that it makes sense and brings the yml file in a uniform order.

### CODE REVIEW

File: _data/training-schools.yml [hunk 1]
```diff
@@ -1,179 +1,177 @@
-- title: AEPSHEP (Asia-Europe-Pacific School of High-Energy Physics)
```
Comment (klieret) [inline _data/training-schools.yml]: Nothing happens in this file, it's just sorted by date and the dictionary keys are sorted alphabetically; all blanks are removed.

**Comment by klieret** [comment]
@GuillermoFidalgo  Can you try if this also works for you: Just execute ``scripts/add_event.py`` and it should guide you through adding a new event

**Comment by GuillermoFidalgo** [comment]
Sure

**Comment by klieret** [comment]
As I have tested the script several times and it doesn't change the webpage itself, I will proceed and merge this; we can still modify/roll back later.

Because it reorders the yml file and I want to add new events (which risks merge conflicts), I don't want to leave this PR pending.

Hope that's alright for everyone.

**Pull request merged**